Fibonacci.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+public class Fibonacci {
+  public static void main(String[] args) { 
+    System.out.println("Enter an integer : "); 
+    Scanner input = new Scanner(System.in);
```

Comment:
if new cannot create Scanner and throws exception then user will see "Enter an integer" and stacktrace, so it's better firstly create scanner and after it print message